src/components/InputArea.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useContext, useEffect, useState } from "react";
+import { Context } from "./Provider";
+
+const InputArea = () => {
+    const { temporaryTodo, increaseTodos, updateTodoText } =
+        useContext(Context);
+
+    const [todo, setTodo] = useState(temporaryTodo);
+
+    const handleTodoChange = (e) => {
+        const { value } = e.target;
+        setTodo({ text: value, id: temporaryTodo.id, checkness: false });
+    };
+    useEffect(() => {
+        setTodo(temporaryTodo);
+    }, [temporaryTodo]);
```

Comment:
You don't need to use the useEffect as the code inside does not have side effects.